components/02-elements/table/_table.scss

```diff
@@ -109,18 +119,34 @@ $table-radius: $border-radius;
     }
 
     tfoot {
-        td { 
-            padding: 5px $table-padding;
-            background: $white;
+        td {
             display: block;
+            padding: 5px $table-padding;
+            background: $table-tfoot-background;
             text-align: left;
 
-            @include mq($screen-m) {
+            @include mq($table-breakpoint-medium) {
+                display: table-cell;
                 padding: $table-padding;
-                background: $gray-lightest;
-                border-right: 1px solid $gray-lighter;
+                background: $table-tfoot-breakpoint-background;
+                border-right: $table-border-inside;
                 text-align: left;
-                display: table-cell;
+            }
+        }
+    }
+
+    &--odd-even {
```

Comment:
Follow the rules - https://docs.snowdog.pro/web/frontend/how-to-write-better-code.html

Reply:
modifiers moved